scripts/flip-files-to-nightly.sh

```diff
@@ -1,13 +1,19 @@
 #!/bin/bash
 
+export VERSION=$(date +'%-y.%-m.%-d')
+export COMMIT=$(git rev-parse --short HEAD)
+
 # package.json
-VERSION=$(date +'%-y.%-m.%-d') yarn files:set-version
+yarn files:set-version
 echo "$(jq --arg name 'Zoo Modeling App (Nightly)' '.productName=$name' package.json --indent 2)" > package.json
 
 # electron-builder.yml
 yq -i '.publish[0].url = "https://dl.zoo.dev/releases/modeling-app/nightly"' electron-builder.yml
 yq -i '.appId = "dev.zoo.modeling-app-nightly"' electron-builder.yml
 
+# Release notes
+echo "Nightly build $VERSION (commit $COMMIT)" > release-notes.md
```

Comment:
[question] Oh cool so does the nightly build get automated release notes in the updater modal then?

Reply:
No the release notes will just be this message, pointing to the version (which is a date YY.MM.DD) and a commit hash